tests/existing-resources/main.tf

```diff
@@ -27,3 +27,9 @@ resource "ibm_is_subnet" "testacc_subnet" {
   total_ipv4_address_count = 256
   resource_group           = module.resource_group.resource_group_id
 }
+
+resource "ibm_resource_instance" "cos_instance" {
+  name              = var.cos_instance_name
+  resource_group_id = module.resource_group.resource_group_id
+  service           = "cloud-object-storage"
+}
```

Comment:
No need to create a COS instance, there is a permanent COS instance we can use for tests -> https://github.com/terraform-ibm-modules/common-dev-assets/blob/9fdee86807c74a35061fc3327c2a01215897cb7f/common-go-assets/common-permanent-resources.yaml#L174